central/signatureintegration/datastore/validate.go

```diff
@@ -2,10 +2,12 @@ package datastore
 
 import (
 	"encoding/pem"
+	"regexp"
 	"strings"
 
 	"github.com/hashicorp/go-multierror"
```

Comment:
Not to fix in this PR but we must be constent so probably ban hashicrop/go-multierror in roxvet
https://github.com/stackrox/stackrox/blob/866d8603c90111d5579b54369b92da8961584569/pkg/errorhelpers/format.go#L10-L15

Reply:
Yeah I think we should at some point.